config/locales/en.yml

```diff
@@ -103,13 +178,170 @@ en:
       start_now: Start now
```

Comment:
we have a few `start_now` keys - under different sections, but all with the same wording.

Reply:
They're under different sections, but I think it makes sense to unify them.